src/types/SortType.ts

```diff
@@ -0,0 +1,5 @@
+export enum SortType {
+  ALPHABET,
+  LENGTH,
+  Default,
```

Comment:
[CODE STYLE] - Enum members should be in PascalCase, meaning each word starts with an uppercase letter and the rest of the word is in lowercase. The member 'Default' should be 'Default', to maintain consistency and follow best practices.